tests/integration-tests/configs/common/common.yaml

```diff
@@ -499,25 +505,23 @@ storage:
     dimensions:
       - regions: ["ap-northeast-1"]
         instances: {{ common.INSTANCES_DEFAULT_X86 }}
-        oss: {{ common.OSS_COMMERCIAL_X86 }}
-        schedulers: ["slurm"]
-      - regions: ["cn-north-1"]
-        instances: {{ common.INSTANCES_DEFAULT_X86 }}
-        oss: {{ common.OSS_CHINA_X86 }}
+        oss: ["centos7"]
         schedulers: ["slurm"]
-      - regions: ["ap-northeast-1", "cn-north-1"]
-        instances: {{ common.INSTANCES_DEFAULT_X86 }}
-        oss: {{ common.OSS_BATCH }}
-        schedulers: ["awsbatch"]
-  test_efs.py::test_existing_efs:
+  # The checks performed in test_efs_same_az is similar to test_multiple_efs.
+  # We should consider this when assigning dimensions to each test.
+  test_efs.py::test_multiple_efs:
     dimensions:
-      - regions: ["ap-northeast-2"]
+      - regions: ["ap-northeast-2", "cn-north-1"]
```

Comment:
Is `cn-north-1` chosen here to spread things out or to test support in `cn-*` regions? If it's the former I'd suggest spreading out to non-`cn-*` regions.

Reply:
To test cn region. While reorganizing the test, I was trying not to reduce the coverage. The cn was tested in the before my change